src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,22 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-
-import { useAuth } from '../../providers/Auth';
+import React from 'react';
 import './Home.styles.css';
+import VideoItem from '../../components/VideoItem/VideoItem';
+import getVideoList from '../../mock/mockedData';
+import styled from 'styled-components';
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+const VideoContainer = styled.section`
+  text-align: center;
+`;
 
+function HomePage() {
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
-    </section>
+    <VideoContainer key={Math.random()}>
+      <h1 key={Math.random()}>Video List</h1>
+        {getVideoList()
+        .map((video, index) => (
```

Comment:
You can destructure your props to make the code cleaner when you use the video item component: https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0